examples/CMakeLists.txt

```diff
@@ -18,4 +18,5 @@ endfunction(ConfigureExample)
 ###################################################################################################
 
 ConfigureExample(STATIC_MAP_EXAMPLE "${CMAKE_CURRENT_SOURCE_DIR}/static_map/static_map_example.cu")
+ConfigureExample(CUSTOM_TYPE "${CMAKE_CURRENT_SOURCE_DIR}/static_map/custom_type.cu")
```

Comment:
If I was going to establish the `BUILD_PASCAL_CODE` compile definition I would do the following:

- Drop the changes in the root level CMakeLists.txt
- Check for the flag in `example/CMakeLists.txt`
- Add the define using `target_compile_definitions(... PRIVATE ... )`

In total the changes to example/CMakeLists.txt would look roughly like:


```cmake
ConfigureExample(CUSTOM_TYPE "${CMAKE_CURRENT_SOURCE_DIR}/static_map/custom_type.cu")

if("60" IN LIST CMAKE_CUDA_ARCHITECTURES)
  target_compile_definitions(CUSTOM_TYPE PRIVATE BUILD_PASCAL_CODE)
endif()
```

Reply:
Revert back to CMake `MATCH` solution since `IN_LIST` can deal with exact matches only.